packages/discord.js/typings/index.d.ts

```diff
@@ -3803,17 +3803,24 @@ export interface AvatarDecorationData {
   skuId: Snowflake;
 }
 
```

Comment:
Any reason why this was moved up? Seems weird to have UserPrimaryGuild in between those two collectible related interfaces...